File: j6helloword-demo/app/src/main/java/com/jelly/myapp2/model/activity/MainActivity.java
package com.jelly.myapp2.model.activity;

import android.view.View;

import com.jelly.myapp2.base.ui.BaseActivity;
import com.jelly.myapp2.databinding.ActivityMainBinding;

public class MainActivity extends BaseActivity {
    private ActivityMainBinding binding;

    @Override
    protected View getLayoutView() {
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        return binding.getRoot();
    }

    @Override
    protected void initListener() {
        super.initListener();
        binding.tvwTest1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

            }
        });
        binding.tvwTest2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

            }
        });
    }

    @Override
    protected void onDestroy() {
        super.onDestroy();
    }
}

File: j2jetpack/module_main/src/main/java/com/jelly/jetpack/module/main/ui/activity/MainActivity.kt
package com.jelly.jetpack.module.main.ui.activity

import android.view.View
import com.alibaba.android.arouter.facade.annotation.Route
import com.jelly.jetpack.module.base.constant.ARouterConstant
import com.jelly.jetpack.module.base.ui.BaseActivity
import com.jelly.jetpack.module.base.utils.ToastUtil
import com.jelly.jetpack.module.main.databinding.MainActMainBinding

@Route(path = ARouterConstant.Main.MAIN)
class MainActivity : BaseActivity() {
    private lateinit var mBinding: MainActMainBinding

    override fun getLayoutView(): View {
        mBinding = MainActMainBinding.inflate(layoutInflater)
        return mBinding.root
    }

    override fun initListener() {
        super.initListener()
        mBinding.tvwTest1.setOnClickListener {
            ToastUtil.showShort("tvwTest1")
        }
    }
}